dipy/io/vtk.py

```diff
@@ -146,15 +146,19 @@ def save_vtk_streamlines(streamlines, filename,
     polydata.SetPoints(vtk_points)
     polydata.SetLines(vtk_lines)
 
-    writer = vtk.vtkPolyDataWriter()
-    writer.SetFileName(filename)
-    writer = utils.set_input(writer, polydata)
+    file_extension = filename.split(".")[-1].lower()
+    if file_extension != ".vtk" or file_extension != ".fib":
```

Comment:
let's be more pythonic!  `if file_extension not in [".vtk", ".fib"]` instead

Reply:
i have made the corresponding change